File: SofiForce.Models/Models/EntityModels/BranchModel.cs
using System;

namespace Models
{
    public class BranchModel
    {
        public int BranchId { get; set; }
        public string BranchNameEn { get; set; }
        public string BranchNameAr { get; set; }
        public string BranchCode { get; set; }
        public bool? IsActive { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public int? DisplayOrder { get; set; }
        public decimal? ExpenseRate { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/BranchModel.ts
import { AppUserBranchModel } from "./appUserBranchModel";
import { ClientModel } from "./clientModel";
import { ItemStoreModel } from "./itemStoreModel";
import { RepresentativeModel } from "./representativeModel";
import { SalesOrderModel } from "./salesOrderModel";
import { StoreModel } from "./storeModel";

export interface BranchModel {
    branchId: number;
    branchNameEn: string;
    branchNameAr: string;
    branchCode: string;
    isActive: boolean | null;
    color: string;
    icon: string;
    displayOrder: number | null;
    expenseRate: number | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    latitude: number | null;
    longitude: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}